clients/apps/web/src/components/Settings/GeneralSettings.tsx

```diff
@@ -61,42 +63,59 @@ const GeneralSettings = () => {
+    <ShadowListGroup>
+      <ShadowListGroup.Item>
+        <div className="flex flex-row items-start justify-between">
+          <div className="flex flex-col gap-y-1">
+            <h3>Theme</h3>
+            <p className="dark:text-polar-500 text-sm text-gray-400">
+              Override your browser&apos;s preferred theme settings
+            </p>
+          </div>
+          {theme === undefined ? (
+            <Spinner />
+          ) : (
+            <DropdownMenu>
+              <DropdownMenuTrigger asChild>
+                <Button className="justify-between" variant="secondary">
+                  <span className="capitalize">{theme}</span>
+                  <ExpandMoreOutlined className="ml-2" fontSize="small" />
+                </Button>
+              </DropdownMenuTrigger>
+              <DropdownMenuContent
+                className="dark:bg-polar-800 bg-gray-50 shadow-lg"
+                align="end"
+              >
+                <DropdownMenuItem onClick={handleThemeChange('system')}>
+                  <span>System</span>
+                </DropdownMenuItem>
+                <DropdownMenuItem onClick={handleThemeChange('light')}>
+                  <span>Light</span>
+                </DropdownMenuItem>
+                <DropdownMenuItem onClick={handleThemeChange('dark')}>
+                  <span>Dark</span>
+                </DropdownMenuItem>
+              </DropdownMenuContent>
+            </DropdownMenu>
+          )}
         </div>
-        {theme === undefined ? (
-          <Spinner />
-        ) : (
-          <DropdownMenu>
-            <DropdownMenuTrigger asChild>
-              <Button className="justify-between" variant="secondary">
-                <span className="capitalize">{theme}</span>
-                <ExpandMoreOutlined className="ml-2" fontSize="small" />
-              </Button>
-            </DropdownMenuTrigger>
-            <DropdownMenuContent
-              className="dark:bg-polar-800 bg-gray-50 shadow-lg"
-              align="end"
-            >
-              <DropdownMenuItem onClick={handleThemeChange('system')}>
-                <span>System</span>
-              </DropdownMenuItem>
-              <DropdownMenuItem onClick={handleThemeChange('light')}>
-                <span>Light</span>
-              </DropdownMenuItem>
-              <DropdownMenuItem onClick={handleThemeChange('dark')}>
-                <span>Dark</span>
-              </DropdownMenuItem>
-            </DropdownMenuContent>
-          </DropdownMenu>
-        )}
-      </div>
-    </div>
+      </ShadowListGroup.Item>
+      <ShadowListGroup.Item>
+        <div className="flex flex-row items-start justify-between">
+          <div className="flex flex-col gap-y-1">
+            <h3>Update email</h3>
+            <p className="dark:text-polar-500 text-sm text-gray-400">
+              Update your account&apos;s current email
+            </p>
+          </div>
+          <Link href="/email-update">
```

Comment:
Ideally, I would just embed the form right here, no need for a separate page. The current email should be profiled on the input form.

Similarly, the confirmation can appear right here instead of redirecting to a new page.

Reply:
Thanks for the reviews! btw can we profile the current email as the placeholder or a text in the input form similar to this? : 
<img width="599" alt="Screenshot 2024-12-03 at 3 36 31 PM" src="https://github.com/user-attachments/assets/b44b4664-3d00-4046-bee9-cd3d8142cac6">

Reply:
Also please confirm if  "Similarly, the confirmation can appear right here instead of redirecting to a new page." means that when the user enters the email and hits `Update`, it replaces the input form with the `/request` msg. When the user hits the URL sent in the email, it shows the `/verify` msg (in place of that input form) and when the user hits the `Update the email` button, the email gets updated and the input form reappears.